.fixtures.yml

```diff
@@ -1,9 +1,9 @@
 fixtures:
   repositories:
     stdlib:
-      repo: "git://github.com/puppetlabs/puppetlabs-stdlib"
+      repo: "git@github.com:puppetlabs/puppetlabs-stdlib.git"
```

Comment:
this works fine for travis (don't ask me how). I think the best approach is to use the git:// url, I will update the file